HPXMLtoOpenStudio/tests/test_hvac_sizing.rb

```diff
@@ -414,12 +419,22 @@ def test_manual_j_residences
       ['laundry'] => [2292, 635],
       ['workshop'] => [3057, 192]
     }
-    space_load_results.each do |space_names, sens_loads|
+    total_htg_loads = space_load_results.values.map { |loads| loads[0] }.sum.to_f
+    total_clg_sens_loads = space_load_results.values.map { |loads| loads[1] }.sum.to_f
+    space_load_results.each do |space_names, loads|
+      est_space_duct_loads_htg = (loads[0].to_f / total_htg_loads * 2561.0)
```

Comment:
I calculated an estimated weighted duct loads here and made two comparison with and without duct loads. It's hard to split the total loads without space duct values, so this is the only best solution I can think of.